Decoders/MPEGDecoder.cpp

```diff
@@ -345,7 +345,8 @@ UInt32 SFB::Audio::MPEGDecoder::_ReadAudio(AudioBufferList *bufferList, UInt32 f
 			break;
 		}
 
-		// The analyzer error about division by zero may be safely ignored, because mChannelsPerFrame is verified > 0 in Open()
+		// The analyzer error about division by zero may be safely ignored with assert, because mChannelsPerFrame is verified > 0 in Open()
+    assert(mFormat.mChannelsPerFrame > 0);
```

Comment:
I think comment + assert is a great way to deal both with human beings reading this code and analyzer (it doesn't show warning anymore).